docs/blog/2017-07-19-creating-a-blog-with-gatsby/index.md

```diff
@@ -0,0 +1,486 @@
+
+The actual React component is fairly trivial, but one important note should be made. It's important that when linking to internal content, i.e. other blog links, that it's always advisable to use `gatsby-link`. This is for a variety of reasons, but one key one is that the `Link` component includes the ability to use a `pathPrefix.` This is useful if this blog will be hosted on something like Github Pages, or an otherwise non-root domain level, which are both very common patterns.
+
+Now this is getting exciting and it feels like we're finally getting somewhere! At this point, we have a fully functional blog generated by Gatsby, with real content authored in Markdown, a blog listing, and the ability to navigate around in the blog. If you run `yarn develop`, `http://localhost:8000` should display a preview of each blog post, and each post title links to the content of the blog post. A real blog!
+
+![Blog listing](./blog-listing.png)
+
+It's now on you to make something incredible with the knowledge you've gained in following along with this tutorial! You can not only make it pretty and style with CSS (or [styled-components][styled-components]!), but you could improve it functionally by implementing some of the following:
+
+- Add a tag listing and tag search page
+  - hint: the `createPages` API in `gatsby-node.js` file is useful here, as is frontmatter
+- adding navigation between a specific blog post and past/present blog posts (the `context` API of `createPages` is useful here), etc.
+
+With our new found knowledge of Gatsby and its API, you should feel empowered to begin to utilize Gatsby to its fullest potential. A blog is just the starting point; Gatsby's rich ecosystem, extensible API, and advanced querying capabilities provide a powerful toolset for building truly incredible, performant sites.
+
+Now go build something great.
+
+![Dream Bigger](./dream-bigger.jpeg)
+
+## Links
+
+- [`@dschau/gatsby-blog-starter-kit`][source-code]
+  - A working repo demonstrating all of the aforementioned functionality of Gatsby
+- [`@dschau/create-gatsby-blog-post`][create-gatsby-blog-post]
+  - A utility and CLI I created to scaffold out a blog post following the predefined Gatsby structure with frontmatter, date, path, etc.
+- [Source code for this blog][blog-source-code]
+  - The source code for this blog, which takes the gatsby-starter-blog-post (previous link), and expands upon it with a bunch of features and some more advanced functionality
+
+[react-dom-server]: https://facebook.github.io/react/docs/react-dom-server.html
+[gatsby-release]: https://www.gatsbyjs.org/blog/gatsby-v1/
+
+[gatsby-plugins]: https://www.gatsbyjs.org/docs/plugins/
+
+[gatsby-plugin-catch-links]: https://www.gatsbyjs.org/docs/packages/gatsby-plugin-catch-links/
```

Comment:
I changed all package links yesterday to /packages/* :-( https://github.com/gatsbyjs/gatsby/pull/1549